sdks/python/apache_beam/testing/benchmarks/cloudml/criteo_tft/criteo.py

```diff
@@ -20,8 +20,11 @@
 from __future__ import division
 from __future__ import print_function
 
-import tensorflow as tf
-import tensorflow_transform as tft
+try:
+  import tensorflow as tf
+  import tensorflow_transform as tft
+except ImportError as e:
```

Comment:
you don't need this try except here. It is needed only in the tests.

Reply:
Not adding it was causing import error that is why added it , I'll remove it.

Reply:
You should import `criteo.py` in tests under a `try` `except` block since `criteo.py` has a dependency on `tft` which might not be installed on all test environments.

Reply:
Yeah I've imported criteo.py in a try block